.vuepress/config.js

```diff
@@ -40,9 +40,14 @@ module.exports = {
               path: "/starport-blog/tutorial/",
               directory: true,
             },
+            // {
+            //   title: "Scavenge",
+            //   path: "/scavenge/tutorial/",
+            //   directory: true,
+            // }, 
             {
-              title: "Scavenge",
-              path: "/scavenge/tutorial/",
+              title: "Starport Scavenge",
+              path: "/starport-scavenge/tutorial/",
```

Comment:
Why not replace the main scavenge tutorial?

Reply:
Good catch, thanks! I've removed the old tutorial

Reply:
thank you!!, should `starport-` be removed as well?